docs/mechanisms/deployment.md

```diff
@@ -33,6 +33,12 @@ Parameters
 The logical name of one or more Auto Scaling Groups to create and
 manage a Deployment Group for in CodeDeploy.
 
+### optional_asg | string,array
+
+The logical name of one or more Auto Scaling Groups to create and
```

Comment:
This first sentence sounds a bit like it creates a new DG for the optional ASGs, but in fact (if I understand the code correctly) all ASGs are added to the same DG.